pkg/exporter/exporter.go

```diff
@@ -27,28 +27,45 @@ import (
 )
 
 type Exporter struct {
-	Username  string
-	Password  string
-	ProjectID string
+	Username    string
+	Password    string
+	ProjectID   string
+	UseAppCreds bool
 }
 
-func New(projectID, username, password string) (*Exporter, error) {
+const REGION_LABEL = "region"
```

Comment:
Why are those constants needed? 🤔

Reply:
They are not mandatory - however the same string was used in several lines so we figured it would be more appropriate to replace them with constants, so one could change them in one place instead of dozens of lines.